docs/content/development/_index.md

```diff
@@ -24,7 +24,7 @@ You can take a look at [our roadmap](https://github.com/guardicore/monkey/projec
 
 ### More exploits! 💥
 
-The best way to find weak spots in a network is by attacking it. The [*Adding Exploits*](./adding-exploits/) page will help you add exploits.
+The best way to find weak spots in a network is by attacking it.
 
 It's important to note that the Infection Monkey must be absolutely reliable. Otherwise, no one will use it, so avoid memory corruption exploits unless they're rock solid and focus on the logical vulns such as Hadoop.
```

Comment:
This section makes no sense without that page. Either remove this completely or add a few lines saying that we're reworking this and we'll add a new guide to add exploits soon. I prefer the latter.